crates/loam-cli/tests/it/build_clients/contracts.rs

```diff
@@ -69,6 +112,9 @@ network-passphrase = "Standalone Network ; February 2017"
 
 [production.contracts]
 hello.workspace = true
+soroban_increment_contract.workspace = false
+soroban_custom_types_contract.workspace = false
+soroban_auth_contract.workspace = false
```

Comment:
I find it kind of odd to call it `workspace` at this point. Don't you think it looks odd here? We are disabling... the workspace?

Maybe it was always a weird thing to call it. But maybe now we should rename this field to something like `build-client`. Thoughts?

Reply:
I changed it to `client` since I cant use `-` in rust variables.